app/src/main/java/com/celzero/bravedns/ui/HomeScreenFragment.kt

```diff
@@ -624,8 +680,7 @@ class HomeScreenFragment : Fragment(R.layout.fragment_home_screen) {
         return VERSION.SDK_INT >= VERSION_CODES.ICE_CREAM_SANDWICH
     }
 
-    @Throws(ActivityNotFoundException::class)
-    private fun prepareVpnService(): Boolean {
+    @Throws(ActivityNotFoundException::class) private fun prepareVpnService(): Boolean {
```

Comment:
Log the exception. `Log.e(tag, msg, ex)`.

Reply:
Added the exception to the log